database/migrations/2024_01_22_233137_prepare_tables_for_next _year.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+return new class extends Migration
+{
+    public function up()
+    {
+        // Create a copy of the 'reservations' table
+        Schema::create('r2024_rezervace', function (Blueprint $table) {
```

Comment:
Copy table like old table
![Screenshot 2024-01-23 at 9 42 35](https://github.com/SU-UTB/web-ples-be/assets/58554772/2e880475-7434-4e7e-bb32-1fdccf2e9fa8)